ocaml/xapi/xapi_clustering.ml

```diff
@@ -165,8 +165,10 @@ module Daemon = struct
     | None -> ignore (Helpers.call_script script params)
 
   let service = "xapi-clusterd"
+  let port = (string_of_int !Xapi_globs.xapi_clusterd_port)
```

Comment:
Careful: the port is defined by xapi startup and any changes to !Xapi_globs.xapi_clusterd_port over the lifetime of xapi will not be taken into account. You might want to put this inside `enable()` and `disable()` to read it again at the time of usage.

Reply:
Indeed, `port` may evaluate before the config file read takes place.